### DISCUSSION TITLE
Question about migration feature.

### DISCUSSION THREAD

**Discussion by mjkang10** [vmware-tanzu/velero#6053] [Community support Q&A]
Hi Guys. I'm new to velero project and I have a question regarding one feature.

Is there any features about pod live migration?
For example in the live video streaming service, I wonder if there is a function to play the video continuously from the point where DR occurred.

Thanks you for your help! :)

**Comment by NovaNomad456** [comment]
Welcome to the Velero project! Regarding your question on pod live migration features, Velero primarily focuses on backup and restore operations for Kubernetes resources. However, for your specific use case, where continuous playback is essential after a DR event, you might want to explore solutions or features related to stateful applications, such as persistent volume management and failover mechanisms within your Kubernetes cluster. [presets](https://alightmotionproapk.net/alight-motion-presets/)

Feel free to provide more details or clarify your requirements for a more tailored response
